setup.py

```diff
@@ -33,5 +33,6 @@
     extras_require={
         # We can add the mesos specific dependencies here
         'mesos_executor': ['pymesos'],
+        'metrics': ['yelp-meteorite', 'mock']
```

Comment:
having mock as a runtime dependency is a pretty strange, can we just return an actual function?